lib/Resque/Job/FactoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+interface Resque_Job_FactoryInterface
+{
+	/**
+	 * @param $className
+	 * @param array $args
+	 * @param $queue
+	 * @return Resque_JobInterface
+	 */
+	public function create($className, $args, $queue);
```

Comment:
@chrisboulton I had to remove the type hint here. As much as the tests are not sending an array here all the time (`null` !== `[]`) I imagine clients would be doing the same.